ice40/tests/iceff/Makefile

```diff
@@ -0,0 +1,3 @@
+ARCH=ice40
+DEVICE=HX1K
```

Comment:
DEVICE=hx1k
Needs to be lowercase now I believe

Reply:
Done.